### DISCUSSION TITLE
Feature Requests

### DISCUSSION THREAD

**Discussion by jswhisperer** [imrofayel/fylepad#20] [Ideas]
Any Feature Requests? or idea

**Comment by jswhisperer** [comment]
I think we should enable PWA for the web version https://pwa.nuxtjs.org/
So users can install the web application to their local systems and have truly offline first experiences

**Reply by imrofayel** [reply]
@jswhisperer It's now available as PWA too, fully responsive for mobile devices.🔮

**Reply by jswhisperer** [reply]
Looks like it's almost there just need some tweaks in the manifest.json
<img width="1153" alt="image" src="https://github.com/user-attachments/assets/5af07b96-dcc2-4c15-8513-9c4ea1ef6c0e" />

**Reply by jswhisperer** [reply]
How are you generating manifest.webmanifest is it on the rust side?

**Reply by imrofayel** [reply]
I haven't change anything on rust side, I used the available icons only cause too lazy to make new ones haha I've not made any manifest file tho just used `vite pwa` config.

**Reply by jswhisperer** [reply]
That's fine, I think you just need the correct icon sizes for the manifest. You can check if its working as per above devtools > application > manifest

**Reply by jswhisperer** [reply]
specifically I think this icon is actually 142x142
<img width="255" alt="Screenshot 2025-04-20 at 4 24 23 am" src="https://github.com/user-attachments/assets/47532bac-2c5f-4087-9008-e6dc24e74529" />

**Reply by jswhisperer** [reply]
actually most the sizes are incorrect currently but the manifest is fine

**Reply by imrofayel** [reply]
I'll resize the icons and fix other issues in `dev tools`.
